Runtime/Plugins/SolanaWalletAdapterWebGL/SolanaWalletAdapterWebGL.jslib

```diff
@@ -155,8 +161,18 @@ mergeInto(LibraryManager.library, {
       var serializedSignedTransactions = [];
       for (var i = 0; i < signedTransactions.length; i++) {
         var signedTransaction = signedTransactions[i];
-        var txStr = signedTransaction.serialize().toString("base64");
-        serializedSignedTransactions.push(txStr);
+        let after = getIx(signedTransaction).length;
+        if (after != instructionIxCounts[i]) {
+          serializedSignedTransactions.push(
+            Buffer.from(signedTransaction.serialize()).toString("base64")
+          );
+        } else {
+          let signature = await asmLibraryArg.GetSignatureForAddress(
+            signedTransaction
+          );
+          let signatureStr = signature ? signature.toString("base64") : "";
+          serializedSignedTransactions.push("s:" + signatureStr);
```

Comment:
in some cases walletadapter does not change the transaction  ( eg: adding compute instructions ) which means we can use same signature for the transaction instead of transmitting the whole transaction, This has been purely done to optimize data sharing b/w WASM runtime and JS.

Reply:
I think the added complexity doesn't justify the optimisation. Consider that all the modern wallets add priority fees to the tx, plus data sharing b/w WASM runtime and JS, I'm not convinced that this implementation is better. Let's revisit based on PR https://github.com/magicblock-labs/Solana.Unity-SDK/pull/234

Reply:
Alright, i will wait for the merge then, and update the PR accordingly